zulipterminal/model.py

```diff
@@ -119,6 +119,17 @@ def __init__(self, controller: Any) -> None:
         self.new_user_input = True
         self._start_presence_updates()
 
+    def get_versions(self) -> Dict[str, str]:
+        """
+        Returns feature level and server version.
+        """
+        settings = self.client.get_server_settings()
+        versions = dict()
+        if settings['result'] == 'success':
+            versions['feature_level'] = str(settings['zulip_feature_level'])
```

Comment:
In the original issue, I was not originally intending that we should show the feature level, as it's just an integer and so not necessarily useful, but *if* it's present (see my note on 'supported server versions` in #**zulip-terminal**) we could include it.